SwiftyStoreKit/SwiftyStoreKit.swift

```diff
@@ -141,7 +141,35 @@ public class SwiftyStoreKit {
             }
         }
     }
+  
+    /**
+    *  Get product info
+    *  - Parameter productId: productId as specified in iTunes Connect
+    */
+    public class func product(for productIdentifier: String?) -> SKProduct? {
+      if let currentProductIdentifier = productIdentifier, let product = sharedInstance.store.products[currentProductIdentifier] {
+        return product
+      }
     
+      return nil
+    }
```

Comment:
The `retrieveProductsInfo()` calls the completion block immediately if the product has already been cached once.

I don't think we need this method, because it would just return a nil product if a prior call to `retrieveProductsInfo()` has not been made (which can be confusing).